src/Common/src/System/Security/Cryptography/Asn1Reader/AsnWriter.BitString.cs

```diff
@@ -120,20 +117,190 @@ private void WriteBitStringCore(Asn1Tag tag, ReadOnlySpan<byte> bitString, int u
             _offset += bitString.Length;
         }
 
-        // T-REC-X.690-201508 sec 9.2, 8.6
-        private void WriteConstructedCerBitString(Asn1Tag tag, ReadOnlySpan<byte> payload, int unusedBitCount)
+#if netcoreapp || uap || NETCOREAPP
```

Comment:
Why do we have two different versions of netcoreapp vs NETCOREAPP?  Can we fix that to just have one compilation constant for it?

Reply:
One of them is from corefx, the other is corefxlab, from when the code gets copied back and forth as part of the Asn1.Experimental package.